#request
float64 Offset
int64 Ausgabestelle
---
#response
int64 status